backend/src/BuiltinCliHost/Utils.fs

```diff
@@ -0,0 +1,147 @@
+module BuiltinCliHost.Utils
+
+open Prelude
+
+module PT = LibExecution.ProgramTypes
+module RT = LibExecution.RuntimeTypes
+module VT = RT.ValueType
+module PT2DT = LibExecution.ProgramTypesToDarkTypes
+module PackageIDs = LibExecution.PackageIDs
+
+open LibExecution.RuntimeTypes
+
+module CliScript =
+  type Definitions =
+    { types : List<PT.PackageType.PackageType>
+      constants : List<PT.PackageConstant.PackageConstant>
+      fns : List<PT.PackageFn.PackageFn> }
+
+  type PTCliScriptModule =
+    { // These will end up in the package manager
```

Comment:
I'm not sure what this comment means

Reply:
me neither, you've added it in https://github.com/darklang/dark/blob/b6a0b2f215ecb116f747f99cb90e634e833c8302/backend/src/LibParser/Canvas.fs#L40-L41

I kept it here just in case it's needed, since Canvas.fs will be deleted at some point

Reply:
whoops. let's remove in both places :)